### PR TITLE
Add message to display command after limits

### PR DESCRIPTION
**Pull request by JitskedeVries**
### Short discription
Add a new message that can be used in the hardware interface to display the commands that are really send towards the iMotionCube. 

### Explanation
First the trajectory is send towards the trajectory controller, this trajectory controller calculates dependently on the error which command must be sent towards the IMC. This calculated command can be seen in several topics. But before the command is written towards the IMC, the limits interface will check if this command is possible in combination with the current position, current speed and all the position, velocity and effort limits. This limit interface adjusts the command before it is send towards the IMC. This new feature makes is possible to see what command is actually sent after the limits.

### Hardware Interface
This message is used in the hardware interface on the feature/realtime-pub-afterlimit-command branch. In the ```void MarchHardwareInterface::write(ros::Duration elapsed_time)``` method in the march_hardware_interface.cpp: https://github.com/project-march/hardware-interface/pull/116

### CODE REVIEW

**Comment by martijnvandermarel** [comment]
@JitskedeVries Can you link to the PR/branch in the hardware interface? Hard to review otherwise

File: march_shared_resources/msg/AfterLimitCommand.msg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+string[] name
+float64[] position_command
```
Comment (martijnvandermarel) [inline march_shared_resources/msg/AfterLimitCommand.msg]: Are the position_command and effort_commands really 64 bit (long floats) or are they normal floats (32-bit)?

Comment (Bart-van-Ingen) [reply march_shared_resources/msg/AfterLimitCommand.msg]: The definition of the commands can be found in the hardware interface and are as such:
```
  std::vector<double> joint_position_command_;
  std::vector<double> joint_velocity_command_;
```

**Comment by Bart-van-Ingen** [comment]
> Message name is not descriptive enough

@Ishadijcks what needs to added then? would you consider AfterLimitControllerCommand sufficient?

**Comment by Ishadijcks** [comment]
`AfterLimitJointCommand` would be better I think. This also implies what the command is send to.

But this whole feature is only to inspect the effects of the SafetyLimits right? I wonder if there is a cleaner way to fix this, like adjusting the desired values before they were published...

File: march_shared_resources/msg/AfterLimitJointCommand.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+string[] name
+float64[] position_command
+float64[] effort_command
```
Comment (Ishadijcks) [inline march_shared_resources/msg/AfterLimitJointCommand.msg]:
```suggestion
float64[] effort_command

```

**Pull request merged**